openedx/core/djangoapps/discussions/docs/decisions/0005-in-context-discussion-course-blocks.rst

```diff
@@ -0,0 +1,39 @@
+Making Linked In-Context Discussions Available to MFEs
+======================================================
+
+
+Status
+------
+
+Proposal
+
+
+Context
+-------
+
+In the ADR 0004 we investigated how to link a discussion to a unit. We
```

Comment:
Could we add a link to the ADR 004?

Reply:
I will make these changes once that ADR is approved. Till then there is no point linking it since it only exists in a PR.